README.md

```diff
@@ -208,3 +208,8 @@ complex execution modes and dynamic shapes. If not specified, all are enabled by
   Setting the parameter `DISABLE_OPTIMIZED_EXECUTION` to "true" in the model
   configuration may help in some cases to avoid these negative interactions
   due to model specific caching and increase multiple instance performance.
+
+* Since PyTorch does not support Tensor of Strings, Triton allows users to pass
+string input(s)/recieve string output(s) using a List of Strings. Because of using
+List instead of Tensor, string I/O is supported only for 1-dimensional inputs/outputs.
+This means batching must be disabled as well for String I/O on TorchScript models.
```

Comment:
I want to do this and the required test coverage for this in a follow up PR since I would like to get these changes in before code freeze. Reshaping for string lists will be different than the way we do it for tensors. 
Related: https://github.com/triton-inference-server/pytorch_backend/pull/60#discussion_r866184080

Reply:
Should say something like "batching is not allowed for TorchScript models with String I/O, `max_batch_size: 0` must be set for such models". It is more about we haven't decided on how to represent batching with respect to Python string.

Reply:
Done